src/main/resources/warnings.json

```diff
@@ -673,7 +673,8 @@
     "url": "https://jenkins.io/security/advisory/2017-04-10/",
     "versions": [
       {
-        "pattern": ".*"
+        "lastVersion": "1.5.2",
+        "pattern": "1[.][012345](|[.-].*)"
```

Comment:
Jenkins itself matches what's installed to the regex. As this also matches 1.5.3, people with that release will still appear affected. This is a common case, you'll find several patterns dealing with this in the file.

A new version will not make this any more correct, people may be running 1.5.3 and we should not tell them they're unsafe.